src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/GetAzureRmSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -0,0 +1,299 @@
+            HelpMessage = "Whether or not to only get the latest backup per database. Defaults to false.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = GetBackupsByResourceIdSet,
+            HelpMessage = "Whether or not to only get the latest backup per database. Defaults to false.")]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter OnlyLatestPerDatabase { get; set; }
+
+        /// <summary>
+        /// Gets or sets the database state to look for (Alive or Deleted).
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ParameterSetName = ServerNameSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 4,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = DatabaseNameSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 4,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = LocationNameSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 4,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = GetBackupsByInputObjectSet,
+            Position = 2,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = GetBackupsByResourceIdSet,
+            Position = 2,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The state of the database whose backups you want to find, Alive, Deleted, or All. Defaults to All")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet(Management.Sql.Models.LongTermRetentionDatabaseState.All, Management.Sql.Models.LongTermRetentionDatabaseState.Deleted, Management.Sql.Models.LongTermRetentionDatabaseState.Live,
+            IgnoreCase = true)]
+        public string DatabaseState { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Database object to get backups for.
+        /// </summary>
+        [Parameter(ParameterSetName = GetBackupByInputObjectSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Database object to get backups for.")]
+        [Parameter(ParameterSetName = GetBackupsByInputObjectSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The Database object to get backups for.")]
```

Comment:
Seems like `database` should not have capital `D`

Reply:
Changed. Will be in next commit.